### PR TITLE
Remove people from teams

### PR DESCRIPTION
**Pull request by teoljungberg**
Companies come and go, and so do their employees. Some companies in the
community have had quite the turmoil in terms of their staffing, and
most the list of employees are out of date.

To mitigate that fact, I am removing the list of employees of each
company. If you want to get in touch with someone, do so over our Chat
or go to a meetup.

### CODE REVIEW

**Comment by teoljungberg** [comment]
This may be a controversial PR, so I'm leaving it open for a while to let people ponder over this change.

File: directory.md [hunk 1]
```diff
@@ -30,447 +26,141 @@ Långt ifrån komplett! [Lägg gärna till](https://github.com/rails-se/rails-se
...
-        {% assign github = 'vasilakisfil' %}{% include github.html %}
-      </li>
-      <li>Dennis Carlsson
-        {% assign github = 'dencarlsson' %}{% include github.html %}
-      </li>
-    </ul>
   </li>
+
   <li>
     <a href="http://www.magplus.com">Mag+</a>
```
Comment (yxhuvud) [inline directory.md]: RIP

**Comment by yxhuvud** [comment]
Considering how extremely out of date it is, this should have been done a long time ago.

**Comment by olleolleolle** [comment]
@teoljungberg LGTM!

**Pull request merged**